config/routes.rb

```diff
@@ -44,7 +44,7 @@
     resources :scheduled_events do
       collection do
         get 'current_events'
-        post 'delete_current_event'
+        get 'delete_current_event'
```

Comment:
basically we are not deleting anything here we are creating a data in eventexception table and hiding the event from calender